pinot-spi/src/main/java/org/apache/pinot/spi/stream/StreamDataServerStartable.java

```
@@ -54,6 +54,22 @@ public interface StreamDataServerStartable {
    */
   void createTopic(String topic, Properties topicProps);
 
+  /**
+   * Delete a data stream (e.g Kafka topic) in the server.
+   *
+   * @param topic
+   */
+  default void deleteTopic(String topic) {
```

minor: can we throw UnsupportedOperationException as we are not adding method for KinesisDataServerStartable

---

Added NotImplementedException